packages/wast-parser/src/tokenizer.js

```diff
@@ -266,7 +266,9 @@ function tokenize(input: string) {
 
       while (
         numberLiterals.test(char) ||
-        (input[current - 1] === "p" && char === "+")
+        (input[current - 1] === "p" && char === "+") ||
```

Comment:
Nice job! As a general observation, would it be possible to unit test this code more directly? Testing it via wat and expected AST produces a lot of additional 'noise'

Reply:
Yes, this is what the new tokenizer tests are for. I aim to split the tests for string->tokens and tokens->ast as much as possible.

Reply:
Yes, that sounds good to me.

Reply:
There are so many cases for number literals :roll_eyes: I will look into generating an exhaustive test set from the spec tests.